SETUP.md

```diff
@@ -35,18 +35,19 @@ Currently, this repository supports the following environments:
 ### Setup Requirements
 
 * Anaconda with Python version >= 3.6. [Miniconda](https://conda.io/miniconda.html) is the fastest way to get started.
```

Comment:
I noticed that this might create some confusion for users on Azure DSVM. This is because by default there is conda installed on a DSVM, installing miniconda will make the the installed conda env pointing to the location under miniconda instead of conda. Probably we want to make it more clear here.